packages/node/src/lib/abort.ts

```diff
@@ -0,0 +1,70 @@
+import { EventEmitter } from 'events'
+
+/**
+ * adapted from: https://www.npmjs.com/package/node-abort-controller
```

Comment:
I try to avoid adding dependencies when it makes sense, as people get touchy about it. IIRC also, mine has built-in typescript support, which the most popular package doesn't have.